### DISCUSSION TITLE
Order by with joins

### DISCUSSION THREAD

**Discussion by euberdeveloper** [prisma/prisma#8350] [Q&A] [answered]
If I have a query that makes  a join, I would like to have an order by on the result of that Join and not on the main table. This does not seem possible, is it?

**Comment by ryands17** [comment, marked as answer]
If the relation is a 1-side relation then you can as shown [here](https://www.prisma.io/docs/concepts/components/prisma-client/filtering-and-sorting#sort-by-relation-preview), otherwise it's not possible and you would need to use a [raw query](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access).

We have a request for this here so it would be great if you could add a 👍 to this so that we can know the priority 🙂 
https://github.com/prisma/prisma/issues/5837

**Reply by euberdeveloper** [reply]
Thank you for the fulmineous answer!!!

I did not know of the preview feature and in my case it solves the problem. I will add the up finger nonetheless!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
